library/src-bootstrapped/scala/runtime/TupledFunction.scala

```diff
@@ -1,160 +1,157 @@
-package scala
-package internal
-
-import scala.runtime.TupleXXL
+package scala.runtime
 
 object TupledFunction {
```

Comment:
Coudl we give this one a different name to avoid confusion, like TupledFunctions?

Reply:
I will change it

Reply:
Done